torch_geometric/nn/conv/message_passing.py

```diff
@@ -434,15 +439,38 @@ def message(self, x_j: Tensor) -> Tensor:
         """
         return x_j
 
+    def explain_message(self, out: Tensor, x_i: Tensor, x_j: Tensor) -> Tensor:
```

Comment:
I would expect that `explain_message` has the default explainability logic integrated, see https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/nn/conv/message_passing.py#L343-L354. `GraphMask` would override this functionality.